ikarus/finiteElements/mechanics/enhancedAssumedStrains.hh

```diff
@@ -338,8 +338,8 @@ namespace Ikarus {
 
       auto strainFunction  = DisplacementBasedElement::getStrainFunction(par);
       const auto& numNodes = DisplacementBasedElement::numberOfNodes;
-      assert(((numNodes == 4 and Traits::mydim == 2) or (numNodes == 8 and Traits::mydim == 3))
-             && "EAS only supported for Q1 or H1 elements");
+      if (not((numNodes == 4 and Traits::mydim == 2) or (numNodes == 8 and Traits::mydim == 3)))
```

Comment:
Can you move this if statement to `setEASType` and check it for the case where `numberOfEASParameters!=0`